index.html

```diff
@@ -168,11 +172,12 @@ <h3>Multi-Post
                   </a>
                 </div>
               </div>
+            </div>
           </div>
         </div>
       </div>
-    </section>
-    <section class="about" id="about">
+    </section> 
+  <section class="about" id="about">
```

Comment:
- [ ] The about section is still missing a few details to match the original design.